hackclub.com.yaml

```
@@ -2304,6 +2304,10 @@ te:
   - ttl: 1
     type: CNAME
     value: cname.vercel-dns.com.
+teach:
+  - ttl: 1
+    type: CNAME
+    value: hackclub.github.io.
```

Per new guidance in README, this TTL is too low, it should be at least 600 seconds

---

```suggestion
teach:
  - ttl: 600
    type: CNAME
    value: hackclub.github.io.
```

For ease of committing :)